### PR TITLE
Add content for the Third year undergraduate courses

### PR DESCRIPTION
**Pull request by Cydonia01**
Also, added former tags for academicians.

### CODE REVIEW

File: data/bouncmpe/people/ozgovde-atay.json [hunk 1]
```diff
@@ -7,7 +7,8 @@
     "former-postdoc",
     "former-cmpe140",
     "cmpe250",
-    "advisor-1"
+    "advisor-1",
+    "former-cmpe350"
```
Comment (doganulus) [inline data/bouncmpe/people/ozgovde-atay.json:11]: Do we know Atay hoca taught CMPE350 in the past?

**Comment by doganulus** [comment]
For each course description, let's add a warning on the top for users, saying that 

```
The information on this page is under construction. 
```

Then, I will merge it.

File: data/bouncmpe/people/ozgovde-atay.json (hunk 1 above)
Comment (Cydonia01) [reply data/bouncmpe/people/ozgovde-atay.json:11]: Ekran görüntüsü 2024-10-13 224408
I saw that in website. Also, there is no info before 2010

**Comment by Cydonia01** [comment]
I added under construction warning to the 3XX en tr pages.

**Comment by Mutti499** [comment]
I have checked and adjusted my own warning files according to yours. But I think this is not a dynamic solution because the word “warning” could be anything in future problems rather an "under construction" message.

**Comment by Cydonia01** [comment]
Added photos to static/images/people directory, because assets/images directory is not added to the public upon running server. Also changed the card template to make the images square.

**Comment by doganulus** [comment]
Let's keep PRs seperate. @Cydonia01 

- Revert your lateet commits on this PR. This PR is only about course content.
- Open a new branch for photos and create a new PR from that branch. 
- The research page will have its own PR, too.

**Comment by Cydonia01** [comment]
I reverted it. Sorry for not notifying before.

**Comment by doganulus** [comment]
Thank you @Cydonia01. Can you rebase your Pr to `main` and update warning tags with my language-aware warning shortcode on the `main` branch. It's called `under-construction-warning`.

**Review by doganulus — APPROVED** [review]

**Pull request merged**